modules/ROOT/pages/_partials/secrets-manager.adoc

```diff
@@ -0,0 +1,715 @@
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-list[secrets-mgr:tls-context:flex-gateway:list]| Lists all Flex Gateway TLS context secrets in a secret group
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-modify[secrets-mgr:tls-context:flex-gateway:modify]| Modifies a Flex Gateway TLS context secret
+|xref:anypoint-cli::secrets-manager.adoc#secret-TLS-flex-replace[secrets-mgr:tls-context:flex-gatway:replace]| Replaces an existing Flex Gateway TLS context secret
+
+
+|===
+
+// end::summary[]
+
+// tag::commands[]
+
+[[secret-group-create]]
+== secrets-mgr:secret-group:create
+
+----
+> secrets-mgr:secret-group:create [flags] 
+----
+
+Creates a new secret group with the name specified in `--name`.
+
+Prompt the `--downloadable` flag if the secrets in this group are referenced in an API Manager proxy.
+
+This command also accepts the default `--help` flag.
+
+
+[[secret-group-delete]]
+== secrets-mgr:secret-group:delete
+
+----
+> secrets-mgr:secret-group:delete [flags] 
+----
+Deletes the secret group specified in `--id` +
+
+[WARNING]
+This command does not prompt twice before deleting. If you send a delete instruction, it does not ask for confirmation.
+
+This command accepts the default flag `--help`.
+
+[[secret-group-describe]]
+== secrets-mgr:secret-group:describe
+
+----
+> secrets-mgr:secret-group:describe [flags]
+----
+
+Returns the details of a secret group specified in `--id`. 
+
+This command accepts the default flag `--help`.
+
+[[secret-group-list]]
+== secrets-mgr:secret-group:list
+
+----
+> secrets-mgr:secret-group:list [flags]
+----
+
+Displays all your secret groups. It returns the name of the secret group, and the ID.
+
+This command accepts only the default flag `--help`.
+
+[[secrets-group-modify]]
+== secrets-mgr:secret-group:modify
+
+----
+> secrets-mgr:secret-group:modify  [flags] 
+----
+
+Modifies a secret group specified in `--id`. +
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--name` | New name for your secret group | `--name TestSecretGroup`
+|`--downloadable` | The secrets in this group are referenced in an API Manager proxy.
+ | `--downloadable`
+|`--no-downloadable` | TThe secrets in this group are not referenced in an API Manager proxy. | `--no-downloadable`
+
+|===
+
+[[secret-shared-create]]
+== secrets-mgr:shared-secret:create
+
+----
+> secrets-mgr:shared-secret:create [flags] 
+----
+
+Creates a new shared secret in the secret group specified in `--group-id`, using the name specified in `--name` and the type specified in `--type`. 
+
+Besides the default flag `--help`, this command also accepts:
+
+[%header,cols="30a,40a,30a"]
+|===
+|Flag | Description |  Example
+|`--group-id`| Mandatory. Secret group ID. | `--group-id 1fec0a49-1551-4199-bfcc-cf0352d0f29d`
```

Comment:
I think Description should lead with the description, rather than "Mandatory".
Secret group ID (required)

Reply:
No period unless all are complete sentences.

Reply:
Mandatory or required?